src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4235,39 +4250,52 @@ namespace Akka.Routing
             public Akka.Util.ISurrogated FromSurrogate(Akka.Actor.ActorSystem system) { }
         }
     }
-    public class SmallestMailboxRoutingLogic : Akka.Routing.RoutingLogic
+    public sealed class SmallestMailboxRoutingLogic : Akka.Routing.RoutingLogic
     {
         public SmallestMailboxRoutingLogic() { }
         public SmallestMailboxRoutingLogic(int next) { }
         public override Akka.Routing.Routee Select(object message, Akka.Routing.Routee[] routees) { }
     }
+    public sealed class TailChopping : Akka.Routing.RoutingLogic
+    {
+        public TailChopping(Akka.Actor.IScheduler scheduler, System.TimeSpan within, System.TimeSpan interval) { }
+        public override Akka.Routing.Routee Select(object message, Akka.Routing.Routee[] routees) { }
+    }
     public sealed class TailChoppingGroup : Akka.Routing.Group
     {
         public TailChoppingGroup(Akka.Configuration.Config config) { }
-        public TailChoppingGroup(string[] routeePaths, System.TimeSpan within, System.TimeSpan interval) { }
+        public TailChoppingGroup(System.Collections.Generic.IEnumerable<string> routeePaths, System.TimeSpan within, System.TimeSpan interval) { }
```

Comment:
Even though this is a public signature change, it's backwards compatible with what we had before. I'm fine with this.